src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1505,15 +1506,13 @@ public void setBedSpawnLocation(@Nullable Location loc, boolean force)
 	@Override
 	public boolean getAllowFlight()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return allowFlight;
 	}
 
 	@Override
 	public void setAllowFlight(boolean flight)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		this.allowFlight = flight;
 	}
```

Comment:
By CraftBukkit's specification, setting this to `false` while `isFlying()` returns `true` will also set `flying` to `false`.

In other words, if you disallow a player from flying - while he is flying - he needs to stop flying.
This behaviour needs to be added here.